js/modules/loki_rpc.js

```diff
@@ -101,10 +102,22 @@ const fetch = async (url, options = {}) => {
 };
 
 // Wrapper for a JSON RPC request
-const rpc = (address, port, method, params, options = {}) => {
+const rpc = (
+  address,
+  port,
+  method,
+  params,
+  options = {},
+  seedRequest = false
```

Comment:
Having the endpoint passed as argument, and default it to `endpointBase` would seem more sustainable, if/when we have additional request types. Thoughts?

Reply:
Ah yes of course, this felt silly but I just wanted to make it work so wasn't thinking very hard